docs/circuit_diagrams/module_PI/README.md

```diff
@@ -0,0 +1,40 @@
+The Raspberry Pi Module
+===============
+
+The **Raspberry PI module** integrates a [Raspberry Pi 3](https://www.raspberrypi.org/products/raspberry-pi-3-model-b/) into the circuit to add another layer of functionality. It is initially intended to include the following features:
+
+*A 7'' Touchscreen to allow for a GUI
+*A timestamped logging system
+*A way to manage users and keycodes
+*Email/Text Message alerts
+*SSH control
+*USB webcam support
+*A web interface
+
+
+
+Parts
+-----
+* [Raspberry Pi 3 Model B](https://www.raspberrypi.org/products/raspberry-pi-3-model-b/)  
+  -[This kit](https://www.amazon.com/LoveRPi-Raspberry-Plug-Play-Starter/dp/B01IYBZEV6/ref=sr_1_16?s=pc&ie=UTF8&qid=1491114514&sr=1-16&keywords=raspberry+pi+3) comes with all the hardware you need to get started.
+* [7 Inch Touchscreen](https://www.amazon.com/Raspberry-Pi-7-Touchscreen-Display/dp/B0153R2A9I/ref=sr_1_1?s=electronics&ie=UTF8&qid=1491114627&sr=1-1&keywords=7+inch+touchscreen+raspberry+pi)
+* [Raspberry Pi Touchscreen Case](https://www.amazon.com/Case-Official-Raspberry-Touchscreen-Display/dp/B01HV97F64/ref=pd_sim_147_1?_encoding=UTF8&psc=1&refRID=HCS99YG62WD3F73SRZXA)
+
+
+Setup
+=====
+Since this not much is required for the Raspberry Pi Module to work, it would be wasteful to use the whole Raspbian Operating System to do what we're trying to do. Instead, what we've done is build the OS up from scratch starting with [Raspbian Jesse Lite](https://www.raspberrypi.org/downloads/raspbian/) and then installing the [PIXEL](https://www.raspberrypi.org/blog/introducing-pixel/) shell by following [this guide](https://www.raspberrypi.org/forums/viewtopic.php?f=66&t=133691). We then installed Java and set up a developer and user account and installed VNC and SSH. 
```

Comment:
Not sure exactly what you are meaning by "Since **this** not much is required". Did you mean it to say "Since not much is required" or possibly "Since there is not much required"?